blackout/parserapp/views.py

```diff
@@ -1,12 +1,51 @@
-# from .services.parser import saving
+from django.shortcuts import render, redirect, get_object_or_404, get_list_or_404
+from django.contrib.admin.views.decorators import staff_member_required
+from .models import Streets, Buildings
 
-# Create your views here.
 
-from django.http import HttpResponse
+@staff_member_required()
+def fetch_cities(request):
+    cities = Streets.objects.order_by('City').values('City').distinct()
+    context = {'cities': cities}
+    return render(request, 'parserapp/cities_list.html', context=context)
 
 
-def start(request):
-    # x = saving()
-    # return HttpResponse(x)
-    None
-    
\ No newline at end of file
+@staff_member_required()
+def fetch_streets(request, city):
+    streets = get_list_or_404(Streets, City=city)
+    context = {'city': city,
+               'streets': streets}
+    return render(request, 'parserapp/streets_list.html', context=context)
+
+
+@staff_member_required()
+def delete_confirmation(request, **kwargs):
+    """
+    Method for confirming deleting objects. Kwargs contains city and street
+    """
+    try:
+        street = kwargs['street']
+    except:
+        street = None
+    context = {'city': kwargs['city'],
+               'street': street}
```

Comment:
`delete_confirmation(request, city, street = None):` using this code you don't need to use kwargs and try except